app/src/views/GrantRegistryNewGrant.vue

```diff
@@ -235,8 +235,11 @@ function useNewGrant() {
     const properties = { websiteURI: website, githubURI: github, twitterURI };
     if (!signer.value) throw new Error('Please connect a wallet');
     if (!isCorrectNetwork.value) throw new Error('Wrong network');
-    const splitLogoURI = (logoURI as string).split('/');
-    const cid = splitLogoURI[splitLogoURI.length - 1];
+    let cid = '';
+    if (logoURI) {
+      const splitLogoURI = (logoURI as string).split('/');
+      cid = splitLogoURI[splitLogoURI.length - 1];
+    }
     const metaPtr = await ipfs
       .uploadGrantMetadata({ name, description, logoPtr: ipfs.formatMetaPtr(cid), properties })
       .then((cid) => ipfs.formatMetaPtr(cid.toString()));
```

Comment:
Two questions:
1. Don't we require an image to be uploaded on our new grant form? So how does `logoURI` end up undefined?
2. I haven't tested it, but the code seems like it'd result in metadata with an empty `logoPtr` field, but really it should have no `logoPtr` field. Curious if you've tested/confirmed this?

Reply:
1) I have tested this out, the grant form does not require a logo image to be uploaded - is this a new behavior we want? 

2) In terms of empty vs no `logoPtr` field, I believe it was always either empty or has a logoURI value

Reply:
Yup it's an optional field which is weird ! Cause ideally we'd want users to upload the image on grant creation.
Worth bringing it up on the call today ?

Reply:
> I believe it was always either empty or filled with a logoURI value

I wanted to confirm this so we don't accidentally introduce a breaking changes, so:
1. First I checked out commit 14a84fd (an arbitrary commit before the `logoPtr` change), manually fixed the IPFS endpoint URL, then created a grant with no logo. The metadata for that can be found [here](https://bafybeiefsz5an7n6y6n7sppnr5qnkjdqjz55qywvjerqtlc2fqmb2hjj4a.ipfs.dweb.link/). You can this **does not have** a `logoURI` field
2. Then I checkout the latest commit on this branch and created a new grant. The metadata for that can be found [here](https://bafybeihib727xtcj5qzhaet4lbne34q4f6bcc3jxpo6d2a3xu2xrmqpice.ipfs.dweb.link/). You can see this **does have** a `logoPtr` field, and the `pointer` value is an empty string

So we don't want to merge this as is, because then we'll have portions of the app that think a valid `logoPtr` exists, and it has a valid `protocol` value of `1`, but the app will error when it tries to resolve that logo because the `pointer` is empty.

To result the current bug this PR is intended to fix we can either:
1. Require a logo be uploaded when creating a grant (new behavior), OR
2. Modify the grant metadata upload here so the full `logoPtr` object is not included in the metadata (the prior behavior)

I'm fine with either approach, we can decide at standup today.

**Second decision**: Interestingly, we can also see that previously we had inconsistent behavior of how the metadata is structured when empty fields are present: we had no `logoURI` field when there was no logo, but we did have present-but-empty `properties`, `websiteURI`, `githubURI`, `twitterURI` even when those fields are empty. We should pick an approach for those also and can fix it in this PR

Reply:
Sounds good, we should discuss this in standup - I'm fine with either solution as well

Reply:
Let's investigate this option:

Don't require a logo. If the user doesn't provide one, upload to

```
{
  "protocol": 0,
  "pointer":  ""
}
```

When loading the logos, use placeholder if protocol is not 1

Reply:
That seems like a fair way to go about it